plugins/c8y_remote_access_plugin/src/lib.rs

```diff
@@ -40,59 +41,73 @@ pub async fn run(opt: C8yRemoteAccessPluginOpt) -> miette::Result<()> {
-        Command::SpawnChild(command) => spawn_child(command, config_dir.tedge_config_root_path()).await,
-        Command::TryConnectUnixSocket(command) => {
-            match UnixStream::connect(UNIX_SOCKFILE).await {
-               Ok(mut unix_stream) => {
-                   eprintln!("sock: Connected to Unix socket at {UNIX_SOCKFILE}");
-                   write_request_and_shutdown(&mut unix_stream, command).await?;
-                   read_from_stream(&mut unix_stream).await?;
-                   Ok(())
-               }
-               Err(_e) => {
-                   eprintln!("sock: Could not connect to Unix socket at {UNIX_SOCKFILE}. Falling back to spawning a child process");
-                   spawn_child(command, config_dir.tedge_config_root_path()).await
-               }
-           }
+        Command::SpawnChild(command) => {
+            spawn_child(command, config_dir.tedge_config_root_path()).await
+        }
+        Command::TryConnectUnixSocket(command) => match UnixStream::connect(UNIX_SOCKFILE).await {
+            Ok(mut unix_stream) => {
+                eprintln!("sock: Connected to Unix socket at {UNIX_SOCKFILE}");
+                write_request_and_shutdown(&mut unix_stream, command).await?;
+                read_from_stream(&mut unix_stream).await?;
+                Ok(())
+            }
+            Err(_e) => {
+                eprintln!("sock: Could not connect to Unix socket at {UNIX_SOCKFILE}. Falling back to spawning a child process");
+                spawn_child(command, config_dir.tedge_config_root_path()).await
+            }
         },
     }
 }
 
-fn declare_supported_operation(config_dir: &Utf8Path) -> miette::Result<()> {
+fn declare_supported_operation(
+    config_dir: &Utf8Path,
+    user: &str,
+    group: &str,
+) -> miette::Result<()> {
     let supported_operation_path = supported_operation_path(config_dir);
     create_directory_with_user_group(
         supported_operation_path.parent().unwrap(),
-        "tedge",
-        "tedge",
+        user,
+        group,
         0o755,
     )
     .into_diagnostic()
     .context("Creating supported operations directory")?;
 
-    create_file_with_user_group(
-        supported_operation_path,
-        "tedge",
-        "tedge",
-        0o644,
-        Some(
-            r#"[exec]
-command = "c8y-remote-access-plugin"
-topic = "c8y/s/ds"
-on_message = "530"
-"#,
-        ),
-    )
-    .into_diagnostic()
-    .context("Declaring supported operations")
+    if supported_operation_path.exists() {
+        change_user_and_group(supported_operation_path.as_std_path(), user, group)
+            .into_diagnostic()
+            .context("Changing permissions of supported operations")
+    } else {
+        create_file_with_user_group(
+            supported_operation_path,
+            user,
+            group,
+            0o644,
+            Some(
+                r#"[exec]
+    command = "c8y-remote-access-plugin"
```

Comment:
The system test "Install/uninstall c8y-remote-access-plugin" is failing for this extra white space ;-), but better to fix the test removing the useless constraint on the `command` keyword having to start the line. 

```suggestion
command = "c8y-remote-access-plugin"
```